src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/activity/LanternMainActivity.java

```diff
@@ -716,34 +709,8 @@ protected void onActivityResult(int request, int response, Intent data) {
         }
     }
 
-    private void setupNotifications() {
-        PendingIntent pendingIntent = PendingIntent.getActivity(this, 0,
-                new Intent(this, LanternMainActivity_.class)
-                .setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP),
-                0);
-
-        mNotificationBuilder
-            .setSmallIcon(R.drawable.status_on_white)
-            .setCategory(NotificationCompat.CATEGORY_SERVICE)
-            .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
-            .setContentTitle(getText(R.string.app_name))
-            .setWhen(System.currentTimeMillis())
-            .setContentIntent(pendingIntent)
-            .setOngoing(true);
-    }
-
```

Comment:
Just removing the notification stuff too since it's redundant. Resolves https://github.com/getlantern/lantern/issues/4235